testsuite/tests/typing-objects/Tests.ml

```diff
@@ -137,8 +137,8 @@ Lines 1-4, characters 0-3:
 2 |   constraint 'a = int
 3 |   method f x = (x : bool c)
 4 | end..
-Error: The abbreviation c is used with parameters bool c
-       which are incompatible with constraints int c
+Error: The abbreviation c is used with parameter(s) bool
+       which are incompatible with constraint(s) int
```

Comment:
I would use "the class name c" instead of "the abbreviation c". Otherwise the error-message is fine, thanks!

Reply:
Thanks! Reflected (actually pointed part comes from `Bad_parameters`)

Reply:
https://github.com/ocaml/ocaml/pull/11569/commits/597b4e6efd68290dcafe173ae46bba7390f4bc54 and https://github.com/ocaml/ocaml/pull/11569/commits/24805ce763d9832de3e3cc2f14f3ccfb1c2bc6cc should resolve it.